csm/plugins/cortx/provisioner.py

```diff
@@ -58,6 +58,7 @@ class ProvisionerPlugin:
     PRVSNR_NETWORK_PARAM_CIP = 'network/cluster_ip'
 
     def __init__(self, username=None, password=None):
+        self.network_config = None
```

Comment:
With this process on every class initialization will set network_config to none. Lets network config to in -memory conf file

Reply:
Saved network config in Conf.